phpunit/class-wp-rest-url-details-controller-test.php

```diff
@@ -150,12 +108,9 @@ public function test_get_items_fails_for_unauthenticated_user() {
 		$response = rest_get_server()->dispatch( $request );
 		$data     = $response->get_data();
 
-		$this->assertEquals( WP_Http::UNAUTHORIZED, $response->get_status() );
+		$this->assertSame( WP_Http::UNAUTHORIZED, $response->get_status() );
```

Comment:
For my own understanding what's the benefit of `assertSame`?

Reply:
`assertEquals` checks the values are the same but not the data types.
`assertSame` checks that both the value and data type are the same.

PHP 7+ is more type strict. WP Core standardized on `assertSame`.